resources/FieldForV4.php

```diff
@@ -160,6 +160,25 @@ public function create_field($field)
 
         do_action('acf/create_field', $field);
     }
+    
+    /*
+     *  update_value()
+     *
+     *  This filter is applied to the $value before it is updated in the db
+     *
+     *  @param  $value - the value which will be saved in the database
+     *  @param  $post_id - the $post_id of which the value will be saved
+     *  @param  $field - the field array holding all the field options
+     *
+     *  @return $value - the modified value
+     */
+    public function update_value( $value, $post_id, $field ) {
```

Comment:
- Please use PSR2 coding standards where possible. So please use $postId instead of $post_id
- You're not using $field, so you don't have to include it.